src/vma/util/vma_stats.h

```diff
@@ -232,11 +234,30 @@ typedef struct {
 	uint32_t		n_rx_zcopy_pkt_count;
 	uint32_t		n_tx_ready_byte_count;
 	socket_counters_t	counters;
+	bitset<MC_TABLE_SIZE>	mc_grp_map;
+
+	void reset() {
+		fd = 0;
+		inode = tcp_state = 0;
+		socket_type = 0;
+		b_is_offloaded = b_blocking = b_mc_loop = false;
+		bound_if = connected_ip = mc_tx_if = (in_addr_t)0;
+		bound_port = connected_port = (in_port_t)0;
+		threadid_last_rx = threadid_last_tx = pid_t(0);
+		n_rx_ready_pkt_count = n_rx_ready_byte_count = n_rx_ready_byte_limit = n_rx_zcopy_pkt_count = n_tx_ready_byte_count = 0;
+		memset(&counters, 0, sizeof(counters));
+		mc_grp_map.reset();
+	};
 } socket_stats_t;
```

Comment:
It is better to memset all the struct except the bistset elemet :
```
memset(this, 0, sizeof(socket_stats_t) - sizeof(mc_grp_map));
mc_grp_map.reset();
```
We just need to add comment that the bitset must be the last elemet in the struct